app/src/main/AndroidManifest.xml

```diff
@@ -5,6 +5,7 @@
 
     <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />
     <uses-permission android:name="android.permission.ACCESS_COARSE_LOCATION" />
+    <uses-permission android:name="android.permission.READ_PHONE_STATE" />
```

Comment:
Do we really need this new permission? should it be added from common SDK? cc @tarigo

Reply:
Required by `TelemetryUtils` to  obtain cellular network type.

Reply:
Hmm, but was it used in the same manner before in TelemetryUtils? Why do we add the permission now? As far as I can see if no permission is present the exception will be just swallowed https://github.com/mapbox/mapbox-sdk-common/blob/137e07ef970156322ecad99d3ea642a60ee0c428/components/telemetry_service/platform/android/src/main/java/com/mapbox/common/TelemetrySystemUtils.java#L154

Reply:
Discussed offline with @tarigo , I think we can just remove it, this is not required field, and we only gather it if user has already granted it for other purposes.